http/data-url.json

```diff
@@ -22,12 +22,24 @@
             "version_added": true,
             "notes": "The maximum size supported is 4GB"
           },
-          "firefox": {
-            "version_added": true
-          },
-          "firefox_android": {
-            "version_added": true
-          },
+          "firefox": [
+            {
+              "version_added": "59",
```

Comment:
Are you sure of this? It looks to me data URL worked long time ago, but their use as top-level URL restricted in 58.